### PR TITLE
Initialization of variables

### PR DESCRIPTION
**Pull request by pfagner**
Add initialization of member variables

### CODE REVIEW

File: src/ipc/local/LocalIPCProxy.h [hunk 1]
```diff
@@ -128,7 +128,7 @@ class FaceliftIPCLocalLib_EXPORT LocalIPCProxyBinder : public IPCProxyBinderBase
     QString m_interfaceName;
```
Comment (jacky309) [inline src/ipc/local/LocalIPCProxy.h:128]: why not here as well ?

File: src/ipc/local/LocalIPCProxy.h [hunk 2]
```diff
@@ -128,7 +128,7 @@ class FaceliftIPCLocalLib_EXPORT LocalIPCProxyBinder : public IPCProxyBinderBase
...
     LocalIPCRequestHandler *m_serviceObject = nullptr;
     QPointer<LocalIPCServiceAdapterBase> m_serviceAdapter;
```
Comment (jacky309) [inline src/ipc/local/LocalIPCProxy.h:130]: and here ?

**Comment by pfagner** [comment]
Actually I not sure that we have default initialization in current form and therefore we have risk that they remain uninitialized and we get UB.

**Comment by bitmouse** [comment]
Is it still valid? If yes, please resolve conflicts and update PR.

**Comment by pfagner** [comment]
This commit is irrelevant.

**Pull request closed without merging**